dolphinscheduler-task-plugin/dolphinscheduler-task-shell/src/main/java/org/apache/dolphinscheduler/plugin/task/shell/ShellTask.java

```diff
@@ -81,7 +81,7 @@ public void init() {
     public void handle(TaskCallBack taskCallBack) throws TaskException {
         try {
             IShellInterceptorBuilder<?, ?> shellActuatorBuilder = ShellInterceptorBuilderFactory.newBuilder()
-                    .properties(ParameterUtils.convert(shellParameters.getLocalParametersMap()))
+                    .properties(ParameterUtils.convert(taskExecutionContext.getPrepareParamsMap()))
```

Comment:
Does this change will loss local parameters?

Reply:
it will not, `taskExecutionContext.getPrepareParamsMap()` including all parameter, such as startup param, global param, and local's

Reply:
If so this PR LGTM, but the method is strange, I will guess if there exists a prepare parameter type.